### DISCUSSION TITLE
Is there a JS/Bot that automatically updates tags to the newest trending tags on your most recent YouTube video?

### DISCUSSION THREAD

**Discussion by TVARecordings** [community/community#61498] [Other Feature Feedback, Questions, & Ideas]
Hello! 

I was wondering if there perhaps is a program/sotware/Java script (Such as from GitHub) we can run on our PC that 24/7 will update your most recent uploaded public video with the at that time popular tags in your country. 

Something like this: 
 Video URL -> Select Country -> Start

Find popular tags in Country -> update Video’s tags section with the popular tags > Wait 1 hour -> repeat

**Comment by LoboMetalurgico** [comment]
Maybe there is a bot that does this, but be careful because it is against youtube policy, your video (and your channel) can be taken down.
